examples/step-60/doc/intro.dox

```diff
@@ -138,15 +142,16 @@ In the embedding space, a standard finite dimensional space `space_dh` is
 constructed on the embedding grid `space_grid` (`space_dh`), using the
 FiniteElement `space_fe`, following almost verbatim what was done in step-6.
 
-We indicate with
+We indicate the discretizations of the spaces $V$ and $Q$ with
 \f[
 V_h = \text{span} \{v_i\}_{i=1}^n
 \f]
-and with
+and
 \f[
 Q_h = \text{span} \{q_i\}_{i=1}^m
 \f]
-where $n$ is the dimension of `space_dh`, while $m$ is the dimension of `embedded_dh`.
+respectively, where $n$ is `spacedim`, the dimension of `space_dh`, and $m$
+is `dim`, the dimension of `embedded_dh`.
```

Comment:
this is false. n and m are the dimensions of the finite dimensional spaces, i.e., n_dofs for V_h and n_dofs for Q_h. dim and spacedim are the dimension of the space where the domains are embedded...